apps/etterlatte-saksbehandling-ui/client/src/components/behandling/revurderingsoversikt/OmgjoeringAvFarskap.tsx

```diff
@@ -0,0 +1,102 @@
+import { lagreRevurderingInfo } from '~shared/api/revurdering'
+import { ApiErrorAlert } from '~ErrorBoundary'
+import { oppdaterRevurderingInfo } from '~store/reducers/BehandlingReducer'
+import styled from 'styled-components'
+import { NavnInput } from '~components/behandling/revurderingsoversikt/NavnInput'
+
+function hentUndertypeFraBehandling(behandling?: IDetaljertBehandling): OmgjoeringAvFarskapInfo | null {
+  const revurderinginfo = behandling?.revurderinginfo
+  if (revurderinginfo?.type === 'OMGJOERING_AV_FARSKAP') {
+    return revurderinginfo
+  } else {
+    return null
+  }
+}
+
+export const OmgjoeringAvFarskap = (props: { behandling: IDetaljertBehandling }) => {
+  const { behandling } = props
+  const omgjoeringAvFarskapInfo = hentUndertypeFraBehandling(behandling)
+  const [naavaerendeFar, setNaavaerendeFar] = useState(omgjoeringAvFarskapInfo?.naavaerendeFar)
+  const [forrigeFar, setForrigeFar] = useState(omgjoeringAvFarskapInfo?.forrigeFar)
+  const [feilmelding, setFeilmelding] = useState<string | undefined>(undefined)
+  const [lagrestatus, lagre] = useApiCall(lagreRevurderingInfo)
+  const redigerbar = hentBehandlesFraStatus(behandling.status)
+  const handlesubmit = (e: FormEvent) => {
+    e.stopPropagation()
+    e.preventDefault()
+    setFeilmelding(undefined)
+    if (
+      !naavaerendeFar ||
+      !forrigeFar ||
+      !naavaerendeFar.fornavn ||
+      !naavaerendeFar.etternavn ||
+      !forrigeFar.fornavn ||
+      !forrigeFar.etternavn
+    ) {
+      setFeilmelding('Du må legge inn forrige og nåværende registert far')
+      return
+    }
+    const revurderingInfo: RevurderingInfo = {
+      type: 'OMGJOERING_AV_FARSKAP',
+      naavaerendeFar: naavaerendeFar,
+      forrigeFar: forrigeFar,
+    }
+    lagre(
+      {
+        behandlingId: behandling.id,
+        revurderingInfo,
+      },
+      () => oppdaterRevurderingInfo(revurderingInfo)
+    )
+  }
+
+  return (
+    <MarginTop>
+      <Heading size="medium" level="2">
+        Hvem var registrert som far før?
+      </Heading>
+      {redigerbar ? (
+        <SkjemaWrapper onSubmit={handlesubmit}>
+          <NavnInput navn={naavaerendeFar} update={(n: Navn) => setNaavaerendeFar(n)} />
+          <Heading size="medium" level="2">
+            Hvem er registrert som far nå?
+          </Heading>
+          <NavnInput navn={forrigeFar} update={(n: Navn) => setForrigeFar(n)} />
+          <Button loading={isPending(lagrestatus)} variant="primary" size="small">
+            Lagre
+          </Button>
+          {isSuccess(lagrestatus) ? <span>Lagret!</span> : null}
+          {isFailure(lagrestatus) ? <ApiErrorAlert>Kunne ikke lagre navn på far</ApiErrorAlert> : null}
+          {feilmelding ? <ApiErrorAlert>{feilmelding}</ApiErrorAlert> : null}
+        </SkjemaWrapper>
+      ) : (
+        <BodyShort>
+          Nåværende far:{' '}
+          <strong>
+            {!!naavaerendeFar
```

Comment:
Er det ryddigere å sjekke om den er undefined? 🤔

Reply:
Kanskje, men kjenner meg alltid tryggare med denne typen [falsy-check](https://medium.com/@chirag.viradiya_30404/whats-the-double-exclamation-sign-for-in-javascript-d93ed5ad8491)